src/lib/draw-and-pick.js

```diff
@@ -206,11 +195,122 @@ export function pickLayers(gl, {
 }
 /* eslint-enable max-depth, max-statements */
 
+export function pickLayersByBoundingBox(gl, {
```

Comment:
@ericsoco I believe @ibgreen means "occluded" by "hidden". So, if an element is completely occluded. Users won't expect to select it by pointing at it, but they will expect to select it by box selecting.